src/SingleImagePackModel.cpp

```diff
@@ -314,6 +304,37 @@ SingleImagePackModel::save()
     }
 }
 
+void
+SingleImagePackModel::remove()
+{
+    // handle account pack deletion.
+    // Sadly we cannot actually delete the pack,
+    // so we just send an empty pack to clear out its information.
+    if (roomid_.empty()) {
+        http::client()->put_account_data(
+          mtx::events::msc2545::ImagePack(), [](mtx::http::RequestErr e) {
+              if (e)
+                  ChatPage::instance()->showNotification(
+                    tr("Failed to update image pack: %1")
```

Comment:
Tiny nit, but maybe say "clear image pack" or so, so that it is obvious we wanted to empty it :)